site/docs/v1/tech/apis/_two-factor-disable-request-body.adoc

```diff
@@ -0,0 +1,15 @@
+==== Request Body
+
+[.api]
+[field]#applicationId# [type]#[UUID]# [optional]#Optional#::
+The Id of the application that has been configured with a Two Factor Method Added email template. Please see the section on using application specific email templates. This Id is also used for events.
```

Comment:
```suggestion
The Id of the application that has been configured with a Two Factor Method Added email template. Please see the section on using link:#XXX[application specific email templates to learn more]. This Id is also used for events.
```

Where's the link? Doesn't make sense to add this until the section is written.

Reply:
A user likely wouldn't know how to use the `applicationId` parameter without a guide/video showing them how to override the Tenant Email Template. I'll make a guide, link that pr to this one, and then provide the link here to the guide.  @mooreds how does that sound?

Reply:
That sounds good. So the plan is to not merge this until the guide is done in a sep PR?

Works for me.

Reply:
@jaywood128

Reply:
Opened the pull request for the guide here: https://github.com/FusionAuth/fusionauth-site/pull/1882 and added a link and more consistent descript for `applicationId` below. @mooreds